en/docs/graphql-api.md

```diff
@@ -0,0 +1,86 @@
+# GraphQL APIs
+
+Explore how you can easily design, develop, and manage GraphQL APIs in Choreo to implement your business API strategy.
+
+!!! note "Enable GraphQL components"
+    GraphQL API is currently available in Choreo as a preview feature.
+
+## What is a GraphQL API?
+
+[GraphQL](https://graphql.org) is a query language for APIs to fetch and persist data. A GraphQL API is a category of APIs that supports this query language.
+
+A GraphQL API is an alternative for a REST API to address the following limitations experienced by REST API consumers:
+
+- A REST API may need to send multiple requests to several endpoints to retrieve all the required data. A GraphQL API can achieve this via a single query.
+
+    For example, a banking application has to fetch the phone number and the current account balance of an account holder. To achieve this, a REST API may need to send three requests to call three endpoints (e.g., `/customers/<id>` to fetch the initial data, `/customers/<id>/contactInfo` to fetch the phone number, and `/customers/<id>/balances` to fetch the current account balance).
+
+    A GraphQL API can retrieve all these details via one query similar to the sample given below:
+
+    ```
+    query {
+        customer (id: "SIL182882762") {
+            contactInfo {
+                phoneNo
+            }
+            balances(current)
+        }
+    }
+    ```
+    In this scenario, a single request by a REST API under-fetches data compared to a single request by a GraphQL API.
+
+- In some scenarios, a REST API can over-fetch data. In the same example given above, a request sent by REST API to the `/customers/<id>/contactInfo` may result in fetching all the contact information in the array such as the phone number, work phone number, address, email, etc., whereas the application only needs the phone number.
```

Comment:
Should we consider replacing "may" with "can"?

Also regarding the second sentence, should we say "In the same example given above, a request sent by **a** REST API . . . "